src/kernel/bitcoinkernel.cpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <functional>
+#include <string>
+
+extern const std::function<std::string(const char*)> G_TRANSLATION_FUN = nullptr;
```

Comment:
In commit "b-cs: Define G_TRANSLATION_FUN in bitcoinkernel.cpp" (9b6296dcd4f0601b6c9ff5958edf385ca12e6d75)

It might be good to add a comment explaining this like "Define G_TRANSLATION_FUN symbol in libbitcoinkernel library so users of the library aren't required to export this symbol"

Reply:
Done!